src/conf/eslint.json

```diff
@@ -6,15 +6,21 @@
     "plugin:flowtype/recommended"
   ],
   "rules": {
+    "prettier/prettier": "error",
     "jasmine/expect-single-argument": 0,
     "jasmine-expect-two-arguments": 1,
     "jasmine/new-line-before-expect": 0,
     "istanbul-prevent-ignore": 1,
     "max-len": ["error", { "code": 120 }],
     "mocha/no-exclusive-tests": "error",
     "prefer-destructuring": ["error", {"object": false, "array": false}],
-    "comma-dangle": ["error", "never"],
-    "indent": ["error", 2, { "SwitchCase": 1 }],
+    "comma-dangle": 0,
+    "function-paren-newline": 0,
+    "implicit-arrow-linebreak": 0,
+    "no-confusing-arrow": 0,
+    "arrow-parens": ["error", "as-needed"],
+    "space-before-function-paren": ["error", "never"],
+    "indent": 0,
```

Comment:
Why the change here? Also wrt `SwitchCase`

Reply:
Disabled since prettier takes care of this now. What do you mean with switch case?